@navikt/core/react/src/stepper/Stepper.tsx

```diff
@@ -94,41 +94,37 @@ export const Stepper: StepperComponent = forwardRef<
           className,
         )}
       >
-        <StepperContext.Provider
-          value={{
-            activeStep,
-            onStepChange,
-            lastIndex: React.Children.count(children),
-            orientation,
-            interactive,
-          }}
-        >
-          {React.Children.map(children, (step, index) => {
-            return (
-              <li
-                className={cl("navds-stepper__item", {
-                  "navds-stepper__item--behind": activeStep > index,
-                  "navds-stepper__item--completed":
-                    React.isValidElement<StepperStepProps>(step) &&
-                    step?.props?.completed,
-                  "navds-stepper__item--non-interactive":
-                    React.isValidElement<StepperStepProps>(step) &&
-                    !(step?.props?.interactive ?? interactive),
-                })}
-                key={index + (children?.toString?.() ?? "")}
+        {React.Children.map(children, (step, index) => {
+          return (
+            <li
+              className={cl("navds-stepper__item", {
+                "navds-stepper__item--behind": activeStep > index,
+                "navds-stepper__item--completed":
+                  React.isValidElement<StepperStepProps>(step) &&
+                  step?.props?.completed,
+                "navds-stepper__item--non-interactive":
+                  React.isValidElement<StepperStepProps>(step) &&
+                  !(step?.props?.interactive ?? interactive),
+              })}
+              key={index + (children?.toString?.() ?? "")}
+            >
+              <span className="navds-stepper__line navds-stepper__line--1" />
+              <StepperContextProvider
+                interactive={interactive}
+                activeStep={activeStep}
+                lastIndex={React.Children.count(children)}
+                currentStep={index}
```

Comment:
I'm wondering if we can come up with a different name for `currentStep`, b.c. it might be confused with `activeStep` at first glance. Maybe "thisStep" or just "index"? Not important, current name is acceptable.

Reply:
👍 Went for "index"